pkg/cmd/kafka/list/list.go

```diff
@@ -185,6 +180,43 @@ func mapResponseItemsToRows(kafkas []kafkamgmtclient.KafkaRequest, selectedId st
 	return rows
 }
 
+func getClusterIdMapFromKafkas(opts *options, kafkas []kafkamgmtclient.KafkaRequest) (map[string]*v1.Cluster, error) {
+	kafkaClusterIds := make([]string, len(kafkas))
+	for _, kafka := range kafkas {
+		if kafka.GetClusterId() != "" {
+			kafkaClusterIds = append(kafkaClusterIds, kafka.GetClusterId())
+		}
+	}
+
+	idToCluster := make(map[string]*v1.Cluster)
+
+	// if no kafkas have a cluster id assigned then we can skip the call to get
+	// the clusters as we dont need their info
+	if len(kafkaClusterIds) > 0 {
+		clusterList, err := clustermgmt.GetClusterListByIds(opts.f, "", "", createSearchString(kafkaClusterIds), 100)
```

Comment:
Two questions:

1. Do we add a cluster mgmt url to `rhoas kafka list` as a flag and the access token
2. To know the size of the request needed we need to make another api call, should this be done?

Reply:
> Do we add a cluster mgmt url to rhoas kafka list as a flag and the access token

Yes please. These flags are to be hidden.

Reply:
> To know the size of the request needed we need to make another api call, should this be done?

No extra API call is needed.

The size should be at maximum the length of the `kafkaClusterIds` slice - we could use this instead of hardcoding 100.

Reply:
> Yes please. Additionally, these flags are to be hidden.

Agh okay nice, I thought it might be strange but that fact they are hidden is okay